docs/reference/commandline/cli.md

```diff
@@ -45,6 +45,7 @@ by the `docker` command line:
   unsuitable for Docker.
 * `DOCKER_RAMDISK` If set this will disable 'pivot_root'.
 * `DOCKER_TLS_VERIFY` When set Docker uses TLS and verifies the remote.
+* `DOCKER_NOTARY` When set Docker uses notary to sign and verify images.
```

Comment:
Agreed `--untrusted` is a bit weird.

Perhaps `--verify-image=true` as a default, then the user would pass `--verify-image=false`?